### ISSUE TITLE
Add a confirmation while deleting backup

### ISSUE THREAD

**Issue by rajeshisnepali** [teejee2008/timeshift#614]
As the title defines, sometimes we can accidentally click "Delete" or delete the useful backup. 

In that case, if there is a confirmation dialog, that would have been great.

Thanks.

**Comment by jolugaju** [comment]
Yes, please, please, please. That would be great. I have made that mistake several times.

**Comment by SecretAgentAgentX** [comment]
Yes Please!
